### DISCUSSION TITLE
A very stiff E-TKT

### DISCUSSION THREAD

**Discussion by sabeechen** [andreisperid/E-TKT#57] [Show and tell]
I though I'd share a recent rebuild of my E-TKT as well as all the little chnages I've made to (imho) improve it.  I had previously printed one in matte PETG, and it worked great, but I wanted to see if making it out of a stiffer filament would improve it.  I've recently been experimenting with some 20% carbon fiber filament and a 0.6mm ruby nozzle [from amazon](https://www.amazon.com/gp/product/B09XPXD1XD/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1) so I thought I'd give that a shot.  Since the 0.6mm nozzle lets you pump out material pretty quick, I reprinted almost everything with:
- 0.2mm layers
- 5 walls
- 5 top/bottom layers
- 50% infill

The parts came out very chunky and noticeably heavier, just great.  I probably need to dry the filament better, or maybe the filament just is that way, because I had a lot of little blobs on some critical mating surfaces and the overhangs came out a little messy.  No deal breakers though.  The filament is quit abrasive so I needed to file and sand down the area under the lid that the carousel rotates against.  Overall though I think it came out looking great, I'm a big fan of how matte the CF makes the results:

![20230623_164308_A80EFB7D](https://github.com/andreisperid/E-TKT/assets/21026774/4ae7be50-6968-4c4f-9481-e6b671483cb4)
![20230623_164309_3F67F506](https://github.com/andreisperid/E-TKT/assets/21026774/b80b2a95-0c45-4578-86f8-741b705ed7ab)
![20230623_164307_747F0154](https://github.com/andreisperid/E-TKT/assets/21026774/636cd9c4-6145-4b33-b5f1-07bf93901687)

I found the SMD LED's to be a little too bright, so I also printed some little hats for them and hot glued them on to act as makeshift diffusers.  I think it gives a much nicer appearance in low light:
![20230623_164310_B0531A1B](https://github.com/andreisperid/E-TKT/assets/21026774/cc9495dd-a943-4be3-ad01-7312c1463f0e)

When printing, the frame flexes noticeably less than it did with standard PETG.  I notice the lettering looks a little more crisp (might just be in my head) but the cut has definitely improved.  The label is barely hanging on at the end of a print and pulls off with very little tearing/stretching of the label tape.  I might try changing it to cut only once since I suspect the subsequent ones are just gumming up the ends.  Generally I think I'd recommend using this filament if you've got a nozzle that can handle the CF abrasion.

Some other notable changes:
 - I use a screw and small spring to tension the lever arm.  This lets me easily release tension to feed rolls manually or remove jams.
 - I swapped the A4988 for a TMC2208 stepper driver.  This makes the carousel motor almost silent and gets rid of the motor whine I had previously, though it did require modifying the firmware to reverse the motor direction.

**Comment by andreisperid** [comment]
Hi Stephen!

Pretty cool. What is the base material on that carbon fiber composite? PETG?

And nice about the less flexing, it makes sense to improve the cutting, since the overall geometry is supposed to be more stable. Cool idea about the LEDs!

Maybe I will try the TMC008 driver, lets see :)

**Comment by sabeechen** [comment]
The base is still PETG.  If you end up trying a TMC2208, make sure you have it set for STEP/DIR mode, since its usually sold in a UART configuration and the little SMD resistors that commonly configure the setting are very difficult to change with its headers already soldered on.
